### ISSUE TITLE
[Enhancement/New control] Color Picker

### ISSUE THREAD

**Issue by CocoCR300** [lepoco/wpfui#294]
**Is your feature request related to a problem? Please describe.**
This library and WPF are missing a ColorPicker control.

**Describe the solution you'd like**
I've been working on porting the [WinUI](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.colorpicker?view=winui-2.8) color picker to WPF for the last few days and I've made a great progress by now, and I would like to contribute to this library with it.

**Describe alternatives you've considered**
...

**Additional context**
...

**Comment by EddyHaigh** [comment]
Is the color-picker completed but missing from the samples gallery?

**Comment by chucker** [comment]
> Is the color-picker completed

No. That PR would have to be reworked before it can get merged.
